duos-api/src/app.module.ts

```diff
@@ -13,7 +13,7 @@ import { PreauthMiddleware } from './auth/preauth.middleware';
 export class AppModule {
   configure(consumer: MiddlewareConsumer) {
     consumer.apply(PreauthMiddleware).forRoutes({
-      path: '*', method: RequestMethod.ALL
+      path: 'ignore/*', method: RequestMethod.ALL
```

Comment:
why was this changed?

Reply:
that was for testing, thanks for catching that